src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -141,7 +175,7 @@ const useGroupSetting = ({navigation, route}) => {
     isLoading,
     setIsLoading,
     handleResLaunchGallery,
-    updateDataEdit
+    loadingUpdate
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `updateDataEdit` function has been removed and replaced with `loadingUpdate`. Ensure that all references to `updateDataEdit` have been properly updated.

```diff
- updateDataEdit
+ loadingUpdate
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->